README.md

```diff
@@ -296,6 +297,9 @@ kubeless:
   port: 8080 # Port of service of Kubeless function
   kubeconfig: "~/.kube/config" # Kubeconfig file to use (only if falcoside is running outside the cluster)
   # minimumpriority: "debug" # minimum priority of event for using this output, order is emergency|alert|critical|error|warning|notice|informational|debug or "" (default)
+
+webui:
+  address: "" # WebUI address, if not empty, WebUI output is enabled
```

Comment:
It would be better to standardize the naming convention with other settings. Moreover,  "web address" is a colloquial term, so URL would be more appropriate 🙏 

```suggestion

webui:
  url: "" # WebUI address, if not empty, WebUI output is enabled
```